### DISCUSSION TITLE
阿里云盘会员从alist下载被限速

### DISCUSSION THREAD

**Discussion by Terminal-Void** [AlistGo/alist#6816] [General]
限速500kbps以下+Aliyun Video Provider只能播放720p
最新版
多线程的话可以下载到15MB/s左右，但是一般的视频播放器不支持多线程下载，播放1080p的视频有时候也会卡顿
<img width="733" alt="image" src="https://github.com/user-attachments/assets/74f46eea-cb5b-4853-9f05-46033a122ac3">

我用alist给2个朋友分享番剧，和这个会有关系吗。有人最近出现了同样的状况吗

**Comment by SeanHeuc** [comment]
阿里对所有第三方使用都限速了，除非多花钱，官方有说明。
alist 内部有多线程下载并内存缓冲的功能， 理论上可以多线程缓冲单一文件来加速，当然内存和cpu会有更大压力，但大部分设备应该能承受， 均速拉到2.5MB/s-5MB/s  如果官方收到大量的投诉后还不改的话，（确实对老svip用户不公平），可以（改代码）开放这个功能

**Reply by bigbingcom** [reply]
我现在是用windows的robocopy来多线程copy的。alist装docker里面，docker是装istoreos里面，istroreos通过hyperv运行。设置RaiDiver通过WebDav虚拟出来磁盘接alist。在cmd窗口里面，使用命令行robocopy（因为我查询下来，只有robocopy是多线程的下载），加入参数/MT开启多线程，/Z好像是实现断点续传，这样下载速度可以达到3.8M/s左右；如果使用 ctrl+C和ctrl+V在资源管理器中进行复制黏贴，速度是 500Kbps。现在想知道你说的 alist内部有多线程下载，内存缓冲功能怎么操作？

**Comment by cinyan10** [comment]
![image](https://github.com/user-attachments/assets/bd90a4c9-071d-4cd6-9be6-59f06774e261)

**Reply by tegrason** [reply]
笑死

**Reply by aa7914** [reply]
挂羊头 卖狗肉， 吃相开始难堪了，

**Reply by sunlyc** [reply]
阿里不存在打脸问题，因为压根就没有脸

**Comment by Smoovsky** [comment]
同，有SVIP，单IP，无分享。也是总今早开始限速的。

**Comment by AspirantH** [comment]
阿里此举是不是虚假宣传了？呼吁p民联合起来告阿里吧，因为互联网p民被这样当韭菜一次一次割不止一次两次了，现在真的对这种互联网普遍违约的经营主体缺少一种快捷有效的联合维权渠道，如果能专设一个这样的p民权益保护受理渠道就好了，否则类似于阿里这种不信守承诺，虚假宣传的套路拉客没有一点违法成本，还会肆无忌惮

**Comment by RedTearsss** [comment]
自己申请一个阿里云盘开发者，alist里用自己的client_key 、client_secret生成token是否能解除限速？

**Reply by tegrason** [reply]
> 自己申请一个阿里云盘开发者，alist里用自己的client_key 、client_secret生成token是否能解除限速？

没用，限速500K

**Comment by Terminal-Void** [comment]
## 喜报 今天晚上22:53 尝试 alist和infuse可以300Mbps下载
![IMG_9412](https://github.com/user-attachments/assets/5d4992fa-f12b-4319-b0f3-4a7ff9a659a6)
RT
不知道是否为普遍情况
1. 可能与我进行的投诉反馈（黑猫&官方渠道）有关
2. 可能与申请成为开发者有关

**Reply by Terminal-Void** [reply]
## 更新 当天25:33分已恢复500k限速

![IMG_9413](https://github.com/user-attachments/assets/be50f530-a6be-4553-a5b2-c32c54f0057f)

**Reply by tegrason** [reply]
> ## 更新 当天25:33分已恢复500k限速
> ![IMG_9413](https://private-user-images.githubusercontent.com/60608543/351815834-be50f530-a6be-4553-a5b2-c32c54f0057f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4OTM4NTksIm5iZiI6MTcyMTg5MzU1OSwicGF0aCI6Ii82MDYwODU0My8zNTE4MTU4MzQtYmU1MGY1MzAtYTZiZS00NTUzLWE1YjItYzMyYzU0ZjAwNTdmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI1VDA3NDU1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIwMWRiZjUwMmJlNzZmOWYyZmNiMzBhZjZkMDJmOWUwODgzMDBiODgxOGY1NDM3MDg3ODQzMTBlY2M1MDE4NjImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.6de9IhEuVFB32OQIAkDx3AS9tM6gJe0ThQyC4O34TtQ)

我这边还是一样限速500kb

**Reply by Terminal-Void** [reply]
# 发现了离谱的规律 7.26
infuse第一次测试可以达到300+Mbps 但过大约2分钟后会被限制成5Mbps
推测与阿里云盘判断第三方下载总流量是否超过会员每月“赠送”的10GB的算法有关
不会是和四舍五入相关的问题吧
难绷

第一次测试RT
![IMG_9418](https://github.com/user-attachments/assets/5cbf9ebe-9577-4a87-b96d-7884d717e8bd)
第二次测试RT
![IMG_9419](https://github.com/user-attachments/assets/c7721b3b-bfea-4fed-8328-861b54ecd39a)

**Comment by banshaobao** [comment]
你用浏览器下载就是走普通用户了
只有官方客户端下载支持会员

**Reply by Terminal-Void** [reply]
理论上确实是这样的
不过我买包年会员的时候还是没有对第三方速度限制的可以跑到300+Mbps，甚至当时可以使用infuse看蓝光原盘还是其卖点之一
然后过了3，4个月突然一个更新限制了第三方速度到500KBps，需要额外购买第三方权益包
就好像您购买的爱奇艺会员突然限制投屏清晰度要求额外付费一样

**Reply by banshaobao** [reply]
> 理论上确实是这样的 不过我买包年会员的时候还是没有对第三方速度限制的可以跑到300+Mbps，甚至当时可以使用infuse看蓝光原盘还是其卖点之一 然后过了3，4个月突然一个更新限制了第三方速度到500KBps，需要额外购买第三方权益包 就好像您购买的爱奇艺会员突然限制投屏清晰度要求额外付费一样

infuse看蓝光原盘用的是阿里云盘官方的接口，这个接口是有校验会员身份的。而且之前对普通用户也不限速，也可以跑到几百兆每秒。你用的这个浏览器下载完全是脱离阿里云盘官方接口和官方下载的，所以只有普通会员的身份，限制是限速500k/s

**Reply by Terminal-Void** [reply]
> > 理论上确实是这样的 不过我买包年会员的时候还是没有对第三方速度限制的可以跑到300+Mbps，甚至当时可以使用infuse看蓝光原盘还是其卖点之一 然后过了3，4个月突然一个更新限制了第三方速度到500KBps，需要额外购买第三方权益包 就好像您购买的爱奇艺会员突然限制投屏清晰度要求额外付费一样
> 
> infuse看蓝光原盘用的是阿里云盘官方的接口，这个接口是有校验会员身份的。而且之前对普通用户也不限速，也可以跑到几百兆每秒。你用的这个浏览器下载完全是脱离阿里云盘官方接口和官方下载的，所以只有普通会员的身份，限制是限速500k/s

即便是VIP，没有购买第三方权益包的话超过10G的话infuse现在也只有5Mbps是这样的

**Reply by banshaobao** [reply]
> > > 理论上确实是这样的 不过我买包年会员的时候还是没有对第三方速度限制的可以跑到300+Mbps，甚至当时可以使用infuse看蓝光原盘还是其卖点之一 然后过了3，4个月突然一个更新限制了第三方速度到500KBps，需要额外购买第三方权益包 就好像您购买的爱奇艺会员突然限制投屏清晰度要求额外付费一样
> > 
> > 
> > infuse看蓝光原盘用的是阿里云盘官方的接口，这个接口是有校验会员身份的。而且之前对普通用户也不限速，也可以跑到几百兆每秒。你用的这个浏览器下载完全是脱离阿里云盘官方接口和官方下载的，所以只有普通会员的身份，限制是限速500k/s
> 
> 即便是VIP，没有购买第三方权益包的话超过10G的话infuse现在也只有5Mbps是这样的

是的。出了第三方权益包以后，限速500K/s。已转战其他云盘

**Comment by CarlosLucky** [comment]
换平台了，要么就大家私有云服务搞起，难怪最近我看硬盘全线涨价了一波

**Reply by bigbingcom** [reply]
你好，你说的“私有云服务”是怎么弄？

**Reply by Terminal-Void** [reply]
在家里组装NAS服务器，安装硬盘，配置文件共享服务。通过DDNS/内网穿透，端口转发，使可以在外网访问家里NAS服务器上的文件

**Reply by bigbingcom** [reply]
哦，这个方便，也更安全，主动，除了家里宽带的上传速度20M好像。就是外面下载，也快不起来，没有阿里云盘这么快。

**Reply by koradioda** [reply]
关键是资源，资源难下载

**Comment by bruce0781** [comment]
![CleanShot 2024-07-27 at 23 27 28@2x](https://github.com/user-attachments/assets/a4e48979-9610-46d7-94f0-74baa8f807f6)
原因这不就找到了吗？必须买第三方权益包然后才有一个月 1024GB的高速第三方下载。捆绑销售 298 一年
![CleanShot 2024-07-27 at 23 28 09@2x](https://github.com/user-attachments/assets/dd597d02-b13e-4f4d-a7d6-75bf32111150)

**Comment by AspirantH** [comment]
吃相太难看，有什么办法制裁吗？3年SVIP白充了

**Reply by qiuyefeihongwu** [reply]
申请退款，黑猫投诉

**Comment by Terminal-Void** [comment]
能不能抓包看一下伪装成官方客户端（无端的想法）

**Comment by vscodev** [comment]


**Reply by psychopasss** [reply]
普通用户试了，就还是500K。

**Reply by Simonzhang8** [reply]
> 普通用户试了，就还是500K。

没开会员还是照样限速，只有SVIP不限速且不消耗高速流量

**Reply by bruce0781** [reply]
SVIP 測試也是限速 512kb/s....

**Reply by Deepseaon** [reply]
> SVIP 測試也是限速 512kb/s....

开权益包用这个可以不跑流量加并发宽松吧感觉，没权益包速度还是不稳

**Comment by Terminal-Void** [comment]
好在我们小城市家宽直接有公网ip，虽然上传只有50Mbps但是看个1080p还是问题不大的，已经在考虑组个all in boom了（

**Comment by bigbingcom** [comment]
> 限速500kbps以下+Aliyun Video Provider只能播放720p 最新版 多线程的话可以下载到15MB/s左右，但是一般的视频播放器不支持多线程下载，播放1080p的视频有时候也会卡顿 <img alt="image" width="733" src="https://private-user-images.githubusercontent.com/60608543/350287887-74f46eea-cb5b-4853-9f05-46033a122ac3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIyMjg2NjgsIm5iZiI6MTcyMjIyODM2OCwicGF0aCI6Ii82MDYwODU0My8zNTAyODc4ODctNzRmNDZlZWEtY2I1Yi00ODUzLTlmMDUtNDYwMzNhMTIyYWMzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI5VDA0NDYwOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM3YTY3MjJiNDJkMDEzMzgwZjNkNmZjMGEyNjE0NmI4YTI5MTAwYjNhNjNlNDJkZDU4NzE0MWE0Yjg5OGMwMmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0._AmipYJI96L2oFZV8FtS-PZekicaaV8nbXNE7-HYcDE">
> 
> 我用alist给2个朋友分享番剧，和这个会有关系吗。有人最近出现了同样的状况吗

你好，你提到的“多线程的话可以下载到15MB/s左右，”，这句话中的“多线程”是怎么实现的？我就是看了你的主题贴后，去网上找可以实现多线程下载的方法。后来有找到robocopy可以实现。但是我在cmd命令行里面进行的robocopy的方法，是下载速度是3.8MB/s。没有达到你说的 15MB/s的速度啊？

**Reply by Terminal-Void** [reply]
（首先我有普通的VIP，不清楚没购买vip的用户可不可以下到这个速度）
使用Motirx 或者 idm 这样的现成的下载器，或者自己配置aria2下载器通过alist中的`复制编码下载链接` / `发送到Aria2` 进行下载
三天之前的话我使用Motrix的64连接数是可以下载到15MB/s，今天的话还没有试过

如果只是下载文件的话有vip的话还是建议使用官方客户端下载应该是不怎么限速的

**Reply by bigbingcom** [reply]
嗯，理解了。谢谢！因为，我alist是自己本地笔记本里面使用Hyper运行istoreos虚拟机里面部署docker，然后RaiDriver中设置WebDav，把alist虚拟出来一个硬盘。在资源管理器里面复制黏贴操作下载的。Motirx或者IDM这些好像只是支持 Http，Https，和FTP协议。Motirx好像还支持磁力链接和torrent种子下载。关于自己配置aria2，我是路由器里面刷了openwrt，这个里面好像自带的aria2，一直有在使用aria2做下载，路由器有一个U盘接口，接USB硬盘，路由器又做下载机。“通过alist中的复制编码下载链接 / 发送到Aria2 进行下载“这个方法，是第一次听说，等一下试试看？因为没有注意到这个选项。我去找一下，找不到的话再来问一下。

**Comment by cey434** [comment]
反正都是一样的，怎么折腾也折腾不过阿里那群狗啊

**Comment by verydemo** [comment]
TV 接口加密了， 也用不了了吧？  有大神解密了么?

**Reply by Simonzhang8** [reply]
我看楼上有人解密了

**Comment by cey434** [comment]
阿里云盘全面限制第三方软件的挂载限速，已经着手准备退出阿里云盘了。



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年8月4日(星期天) 下午5:52
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] 阿里云盘会员从alist下载被限速 (Discussion #6816)














 
TV 接口加密了， 也用不了了吧？  有大神解密了么?

**Reply by Deepseaon** [reply]
还有什么推荐吗，百度的api也小气，看视频都不方便，115我到搞了一个，但是感觉最近115也是不干人事，真就只能私有云了？但是有些用途还是有个网盘比较方便吧

**Comment by kaneki-029** [comment]
所以使用原版的 阿里云盘 驱动有没有问题，不使用官方 openapi 开发的驱动。

**Reply by bruce0781** [reply]
你是說 AliyundriveOpen 上面那個 Aliyundrive 嗎？那個似乎已經無法使用了。

**Reply by Simonzhang8** [reply]
那个速度更慢好像

**Reply by bruce0781** [reply]
而且好似 Bug 很多...

**Comment by bruce0781** [comment]
第三方權益包還降價了今天，
<img width="811" alt="image" src="https://github.com/user-attachments/assets/7ac98a89-9b04-4e34-aeb3-15b631795c4f">

**Reply by banshaobao** [reply]
没有降价。你是叠加了Kikoeru的优惠券
从第三方跳转到阿里云盘会叠加一张优惠券。使用其他第三方应用也一样

**Reply by ZhangWafer** [reply]
大佬，问一下，如果只买了第三方权益，但是没有开svip，下载速度是快的还是慢的？

**Reply by Deepseaon** [reply]
> 大佬，问一下，如果只买了第三方权益，但是没有开svip，下载速度是快的还是慢的？

那就是你用第三方以及webdav这种方式下载是满速，官方客户端是限速

**Comment by jacknnnbauer** [comment]
充SVIP，用官方的阿里云盘客户端下载还会限速吗？

**Reply by Simonzhang8** [reply]
这倒不会

**Reply by qiuyefeihongwu** [reply]
跑不满，别用阿里，不然有你后悔的

**Comment by WanWuDaQian** [comment]
上周alist挂载阿里云盘，webdav看动画还没问题呢。。。今天一看就卡的不行，登录云盘看到第三方权益包顿感不妙。。。进来一看。芜湖完蛋

**Reply by R1chardRui** [reply]
我是今天限速的，啥都看不成，半秒一卡
